templating/embedding_controllers.rst

```diff
@@ -98,4 +98,36 @@ string syntax for controllers (i.e. **bundle**:**controller**:**action**):
             ) ?>
         </div>
 
+If your controller should be used :doc:`as a service </controller/service>`,
+you can reference it like this instead:
```

Comment:
I tried helping @marcverney set it up on via Slack, but it did not work for him. @marcverney , can you comment on your setup and on the error message you were getting?